bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -777,6 +777,7 @@ EntryLogMetadataMap getEntryLogMetaMap() {
 
     public GarbageCollectionStatus getGarbageCollectionStatus() {
         return GarbageCollectionStatus.builder()
+            .ledgerDir(ledgerDirsManager.getAllLedgerDirs().get(0).getParent())
```

Comment:
For DbLedgerStorage, each directory has on gc thread, and the ledgerDirManager only has one directory. It's OK to get the first directory.  However, for SortedLedgerStorage or InterleavedLedgerStorage, the ledgerDirsManagers has multiple directories, it will lose the other ledgers if we only get the first one.

Reply:
Nope, SortedLedgerStorage and InterleavedLedgerStorage are not supported multiple ledgers yet.